.github/workflows/main.yml

```diff
@@ -141,7 +142,7 @@ jobs:
       - name: Build message
         id: msg
         run: |
-          echo "Preview your changes [here](https://htmlpreview.github.io/?https://github.com/seL4/website_pr_hosting/blob/PR_${{ github.event.number }}/index.html)" > msg.txt
+          # echo "Preview your changes [here](https://htmlpreview.github.io/?https://github.com/seL4/website_pr_hosting/blob/PR_${{ github.event.number }}/index.html)" > msg.txt
```

Comment:
This line causes `msg` to be empty when the link checker doesn't find any issues. We need to add an else and say everything is okay or something.

Reply:
The job should only be running when either the link checker or the validator have found an issue. Will have a look.